Mathlib/RingTheory/TensorProduct/Basic.lean

```diff
@@ -161,6 +161,62 @@ theorem baseChange_neg : (-f).baseChange A = -f.baseChange A := by
 
 end Ring
 
+section liftBaseChange
+
+variable {R M N} (A) [CommSemiring R] [CommSemiring A] [Algebra R A] [AddCommMonoid M]
+variable [AddCommMonoid N] [Module R M] [Module R N] [Module A N] [IsScalarTower R A N]
+
+/--
+If `M` is an `R`-module and `N` is an `A`-module, then `A`-linear maps `A ⊗[R] M →ₗ[A] N`
+correspond to `R` linear maps `M →ₗ[R] N`.
+-/
```

Comment:
This is a definition, not a theorem: so the docstring should tell me _what_ the correspondence is, not just its type!